<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Launcher Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>-30</y>
     <width>331</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget {
    border:none;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="global">
    <attribute name="title">
     <string>Общие</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Путь к игре:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gamePath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browseButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>79</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать...</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cacheCheck">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Чистить кэш при запуске</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="wtfCheck">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Чистить wtf при запуске</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>191</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Время обновления(мсек):</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="updateTime">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>345</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="value">
      <number>300000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Раздел форума 1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Раздел форума 2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Раздел форума 3:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="forum1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="forum2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="forum3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Отображаемые розделы форума</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="clearOnlineButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить макс. онлайн</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Интерфейс</string>
    </attribute>
    <widget class="QCheckBox" name="exitCheck">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Закрыть после запуска WoW</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="realms">
    <attribute name="title">
     <string>Миры</string>
    </attribute>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>360</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="realmsTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="leftWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#leftWidget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(19,19,19), stop:1 rgb(50,50,50));
	border: 1px solid rgb(0,0,0);
}

QPushButton{
	font: 8pt &quot;Arial&quot;;
	color: white;
	border:none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(19,19,19), stop:1 rgb(50,50,50));
	border-bottom: 1px solid rgb(0,0,0);
}

QPushButton:hover {
	 background-color: rgb(76, 76, 76);
}</string>
   </property>
   <widget class="QPushButton" name="generalButton">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>69</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
image: url(:/img/medal.png);
border-left: 3px solid rgb(221,75,57);</string>
    </property>
    <property name="text">
     <string>



General</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guiButton">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>71</y>
      <width>69</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/photo.png);</string>
    </property>
    <property name="text">
     <string>



Interface</string>
    </property>
   </widget>
   <widget class="QPushButton" name="realmsButton">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>142</y>
      <width>69</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/globe.png);</string>
    </property>
    <property name="text">
     <string>



Realms</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
